app/src/main/java/com/gdgevents/gdgeventsapp/ui/theme/Type.kt

```diff
@@ -2,33 +2,86 @@ package com.gdgevents.gdgeventsapp.ui.theme
 
 import androidx.compose.material3.Typography
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import com.gdgevents.gdgeventsapp.R
 
-// Set of Material typography styles to start with
-val Typography = Typography(
+val ProductSans = FontFamily(
+    Font(R.font.product_sans_regular, FontWeight.Normal),
+    Font(R.font.product_sans_bold, FontWeight.Bold)
+)
+
+val AppTypography = Typography(
+    // Headlines
+    headlineLarge = TextStyle(
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Bold,
+        fontSize = 24.sp,
+        lineHeight = 28.8.sp // 120% of font size
+    ),
+    headlineMedium = TextStyle(
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Bold,
+        fontSize = 20.sp,
+        lineHeight = 24.sp
+    ),
+    headlineSmall = TextStyle(
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Bold,
+        fontSize = 18.sp,
+        lineHeight = 21.6.sp
+    ),
+    // Body
     bodyLarge = TextStyle(
-        fontFamily = FontFamily.Default,
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Normal,
+        fontSize = 18.sp,
+        lineHeight = 25.2.sp // 140%
+    ),
+    bodyMedium = TextStyle(
+        fontFamily = ProductSans,
         fontWeight = FontWeight.Normal,
         fontSize = 16.sp,
-        lineHeight = 24.sp,
-        letterSpacing = 0.5.sp
-    )
-    /* Other default text styles to override
-    titleLarge = TextStyle(
-        fontFamily = FontFamily.Default,
+        lineHeight = 20.8.sp
+    ),
+    bodySmall = TextStyle(
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Normal,
+        fontSize = 14.sp,
+        lineHeight = 18.2.sp
+    ),
+    // Buttons
+    labelLarge = TextStyle(
+        fontFamily = ProductSans,
         fontWeight = FontWeight.Normal,
-        fontSize = 22.sp,
-        lineHeight = 28.sp,
-        letterSpacing = 0.sp
+        fontSize = 20.sp,
+        lineHeight = 24.sp
+    ),
+    labelMedium = TextStyle(
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        lineHeight = 20.sp
     ),
     labelSmall = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.Medium,
-        fontSize = 11.sp,
-        lineHeight = 16.sp,
-        letterSpacing = 0.5.sp
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Normal,
+        fontSize = 14.sp,
+        lineHeight = 16.sp
+    ),
+    // Fields
+    displaySmall = TextStyle( // Placeholder
+        fontFamily = ProductSans,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        lineHeight = 24.sp
```

Comment:
Wrong usage of Typography, please refer to material 3 to know which component uses which TextStyle